### ISSUE TITLE
Unable to render custom component in Properties Panel

### ISSUE THREAD

**Issue by billbarni** [bpmn-io/bpmn-js-examples#172]
I am unable to render custom components in my Properties Panel.

The EntryFactory does not exist anymore and I can't seem to find a way to render custom html inside the properties panel.

For some reason I get several errors like not finding "React" to saying that the object being rendered is [Object] instead of a true component.
I've tried a LOT of examples that just return some DIV or something like that with no success.

Is this possible? Is there any example?

**Comment by marstamm** [comment]
Hi,

Did you already have a look at the extension example https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension ?

As this is a new Version of the Properties Panel and the changes are [documented as breaking changes](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/CHANGELOG.md#100), we do not consider this a bug. Please refer to our [Forum](https://forum.bpmn.io/) for general usage questions.

**Comment by billbarni** [comment]
Yes, but I can't implement a custom component. Just the "TextFields" and other existing entries.
How one proceeds to render a Calendar for example in the properties panel?

**Comment by billbarni** [comment]
Using the https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-async-extension and doing:

function Spell(props) {
  const { element, id } = props;

  const modeling = useService('modeling');
  const translate = useService('translate');
  const debounce = useService('debounceInput');

  const getValue = () => {
    return element.businessObject.spell || '';
  }

  const setValue = value => {
    return modeling.updateProperties(element, {
      spell: value
    });
  }

  return (<h1>TEST</h1>)
}

worked for me.

Sorry for opening this ticket and losing your time. Thanks for your patience.
